airflow/dags/download_gtfs_schedule_v2/download_schedule_feeds.py

```diff
@@ -167,39 +162,11 @@ def download_all(task_instance, execution_date, **kwargs):
                 str(f.exception) or str(type(f.exception)) for f in result.failures
             ),
         )
-        # use pandas begrudgingly for email HTML since the old task used it
-        html_report = pd.DataFrame(f.dict() for f in result.failures).to_html(
-            border=False
-        )
-
-        html_content = f"""\
-    NOTE: These failures come from the v2 of the GTFS Schedule downloader.
-
-    The following agency GTFS feeds could not be extracted on {start.to_iso8601_string()}:
-
-    {html_report}
-    """
-    else:
-        html_content = "All feeds were downloaded successfully!"
-
-    if is_development():
-        print(
-            f"Skipping since in development mode! Would have emailed {len(result.failures)} failures."
-        )
-    else:
-        send_email(
-            to=[
-                "laurie.m@jarv.us",
-                "andrew.v@jarv.us",
-                "evan.siroky@dot.ca.gov",
-                "hunter.owens@dot.ca.gov",
-                "jameelah.y@jarv.us",
-                "olivia.ramacier@dot.ca.gov",
-            ],
-            html_content=html_content,
-            subject=(
-                f"Operator GTFS Errors for {datetime.datetime.now().strftime('%Y-%m-%d')}"
-            ),
+        task_instance.xcom_push(
```

Comment:
`xcom` 😅

Reply:
I think this is the appropriate use case 😆